src/components/TemurinDownloadTable/index.tsx

```diff
@@ -16,7 +17,10 @@ const TemurinDownloadTable = ({results}) => {
     return (
     <>
         {source &&
-            <p><Link to={`/temurin/release-notes?version=${source.release_name}`}><MdNotes /> <Trans>Release Notes</Trans></Link></p>
+            <p className='release-notes'>
+                <Link to={`/temurin/release-notes?version=${source.release_name}`} style={{paddingRight: '20px'}}><MdNotes /><Trans>See Release Notes</Trans></Link> 
+                <Link to="/installation/"><IoMdHelpBuoy /><Trans>Read how to install Eclipse Temurin™</Trans></Link>
```

Comment:
Typically you only use the ™ symbol the first time it's mentioned on a page

```suggestion
                <Link to="/installation/"><IoMdHelpBuoy /><Trans>Read how to install Eclipse Temurin</Trans></Link>
```